src/App.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import Calculator from './components/calculator/Calculator';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from './pages/Home';
 
 const App = () => (
-  <div className="App">
-    <Calculator />
-  </div>
+  <Router basename={process.env.PUBLIC_URL}>
+    <Home />
+  </Router>
```

Comment:
- [ ] I think there is no need of a Router here. YOu already have done all the routing in your `Home` component. Kindly remove Router form here.

Reply:
Solved!